### PR TITLE
adds flaps wings emote to moths

### PR DESCRIPTION
**Pull request by lanse12**
## About the PR
added a "flaps wings" emote to moths and the sound for it

## Why / Balance
why not

## Media

https://github.com/user-attachments/assets/7e60110f-4b00-43d8-8739-d7ec9095339e

**Changelog**
:cl:
- add: Moths can now flap their wings.

### CODE REVIEW

**Comment by HenhenIII** [comment]
Time to start using my dead character "Mista cloth" again

**Comment by JoeHammad1844** [comment]
sounds like he broke a bone.
also don't forget aflap

**Review by Aidenkrz — CHANGES_REQUESTED** [review]
all of this needs to have goob comments or be in goob namespace

**Comment by lanse12** [comment]
yea i think this is what you asked me to do

File: Resources/Audio/Voice/Moth/attributions.yml [hunk 1]
```diff
@@ -6,4 +6,9 @@
 - files: ["moth_laugh.ogg, moth_chitter.ogg, moth_squeak.ogg"]
   license: "CC-BY-SA-3.0"
   copyright: "Taken from https://github.com/BeeStation/BeeStation-Hornet/commit/11ba3fa04105c93dd96a63ad4afaef4b20c02d0d"
-  source: "https://github.com/BeeStation/BeeStation-Hornet/blob/11ba3fa04105c93dd96a63ad4afaef4b20c02d0d/sound/emotes/"
\ No newline at end of file
+  source: "https://github.com/BeeStation/BeeStation-Hornet/blob/11ba3fa04105c93dd96a63ad4afaef4b20c02d0d/sound/emotes/"
+
+- files: ["moth_flap.ogg"] #goob thingy
```
Comment (Aidenkrz) [inline Resources/Audio/Voice/Moth/attributions.yml:11]: no reason to be here, should be in goob files, as well as with the actual audio file

File: Resources/Locale/en-US/chat/emotes.ftl [hunk 2]
```diff
@@ -30,6 +30,7 @@ chat-emote-name-monkeyscreeches = Monkey Screeches
 chat-emote-name-robotbeep = Robot
 chat-emote-name-yawn = Yawn
 chat-emote-name-snore = Snore
+chat-emote-name-flap = Flap
```
Comment (Aidenkrz) [inline Resources/Locale/en-US/chat/emotes.ftl:33]: still needs to be in goob files

File: Resources/Prototypes/Voice/speech_emotes.yml [hunk 3]
```diff
@@ -183,6 +183,27 @@
    - squeaks
    - squeaked
 
+- type: emote #goob thingy
```
Comment (Aidenkrz) [inline Resources/Prototypes/Voice/speech_emotes.yml:186]: should be in a goob file

**Review by Aidenkrz — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Voice/speech_emote_sounds.yml [hunk 4]
```diff
@@ -353,12 +353,8 @@
       path: /Audio/Voice/Moth/moth_chitter.ogg
     Squeak:
       path: /Audio/Voice/Moth/moth_squeak.ogg
-    Weh:
```
Comment (Aidenkrz) [inline Resources/Prototypes/Voice/speech_emote_sounds.yml:356]: Why are you removing a ton of emotes here?

File: Resources/Audio/Voice/Moth/attributions.yml [hunk 5]
```diff
@@ -6,4 +6,4 @@
 - files: ["moth_laugh.ogg, moth_chitter.ogg, moth_squeak.ogg"]
   license: "CC-BY-SA-3.0"
   copyright: "Taken from https://github.com/BeeStation/BeeStation-Hornet/commit/11ba3fa04105c93dd96a63ad4afaef4b20c02d0d"
-  source: "https://github.com/BeeStation/BeeStation-Hornet/blob/11ba3fa04105c93dd96a63ad4afaef4b20c02d0d/sound/emotes/"
```
Comment (Aidenkrz) [inline Resources/Audio/Voice/Moth/attributions.yml:9]: dunno what you changed here but it needs to be how it originally was

**Review by Aidenkrz — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Voice/speech_emote_sounds.yml (hunk 4 above)
Comment (lanse12) [reply Resources/Prototypes/Voice/speech_emote_sounds.yml:356]: oh that was the wrong file lmao

File: Resources/Audio/Voice/Moth/attributions.yml (hunk 5 above)
Comment (lanse12) [reply Resources/Audio/Voice/Moth/attributions.yml:9]: it's the same as it originally was

**Review by Piras314 — APPROVED** [review]

**Pull request merged**